includes/payment-methods/class-upe-payment-method.php

```diff
@@ -241,6 +248,15 @@ public function get_icon( string $account_country = null ) {
 		return isset( $this->icon_url ) ? $this->icon_url : '';
 	}
 
+	/**
+	 * Returns icon to use on dark themes.
+	 *
```

Comment:
PHP linter wants a doc comment for the `$account_country` param.

Reply:
Fixed in e69f2320c4fc7effb4ebba2cab5fbd442b0d96a5. 👍